main.go

```diff
@@ -69,10 +72,24 @@ func main() {
 		os.Exit(1)
 	}
 
+	rootSecretName, ok := os.LookupEnv("VAULT_SECRET_NAME")
+	if !ok {
+		setupLog.Error(errors.New("VAULT_SECRET_NAME env variable is not set"), "can't find vault secret name")
```

Comment:
Let's use `"github.com/pkg/errors"` everywhere in project

Reply:
64d17e2